lib/transport/factory.js

```diff
@@ -37,28 +37,25 @@ module.exports = class TransportFactory {
     const {settings, argv} = nightwatchInstance;
```

Comment:
I think argv is always defined, but you could say `const {settings, argv = {}} = nightwatchInstance;` to avoid all those checks below.

Reply:
Yes you are right, but for test cases sometimes `argv` is not passed. I will change it in the suggested manner.

Reply:
for which test cases? can you give an example?